### DISCUSSION TITLE
Getting "Unable to submit your application. Please try again" even if I submitted valid student id and email address

### DISCUSSION THREAD

**Discussion by bassyc** [community/community#73395] [GitHub Education]
### Hi everyone,

Getting "Unable to submit your application. Please try again" even if I submitted valid student id and email address.

Anyone faced similar issue? What can I do to resolve this please?

**Comment by GigaSmurf** [comment]
Without much context, I can only suggest checking your internet, using different browsers or devices, or maybe double check the form for specifics requirements.
